src/refactor_nrepl/find/find_macros.clj

```diff
@@ -61,14 +66,14 @@
               (recur macros (reader/read opts rdr)))))))))
 
 (defn- get-cached-macro-definitions [f]
-  (when-let [[ts v] (get @cache (.getAbsolutePath f))]
+  (when-let [[ts v] (get @macro-defs-cache (.getAbsolutePath f))]
```

Comment:
not sure I follow. I only renamed the `macro-defs` atom also more or less in sync with master. so this is broken on master too?